### PR TITLE
Branch3

### PR DESCRIPTION
**Pull request by swaraj7**

### CODE REVIEW

File: mysite/settings.py [hunk 1]
```diff
@@ -0,0 +1,256 @@
+"""
```
Comment (vishesh92) [inline mysite/settings.py]: There is something very wrong here. Please check.

**Comment by vishesh92** [comment]
```
./imdb/urls.py:9:1: W293 blank line contains whitespace
./imdb/urls.py:9:1: W391 blank line at end of file
```

File: mysite/settings.py (hunk 1 above)
Comment (swaraj7) [inline mysite/settings.py]: this was already present in settings.py. should i remove it ?

File: imdb/movie_rater.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    try:
+        br.open(url)
+        link = br.find_link(url_regex=re.compile(r'/title/tt.*'))
+        res = br.follow_link(link)
+    except:
+        print "wrong movie name"
+        return ""
+    else:
+        soup = BeautifulSoup(res.read())
+        return soup
```
Comment (vishesh92) [inline imdb/movie_rater.py]: Is `soup` a type of `string`?

File: mysite/settings.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+USE_L10N = True
+
+USE_TZ = True
+
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/1.9/howto/static-files/
+
+STATIC_URL = '/static/'
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```
Comment (vishesh92) [inline mysite/settings.py]: `BASE_DIR` is already defined above.

File: requirements.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+BeautifulSoup==3.2.1
+dj-database-url==0.4.1
```
Comment (vishesh92) [inline requirements.txt]: why do you need `dj-database-url`?

Comment (swaraj7) [inline requirements.txt]: dj-database-url is a utility that has been used for database purposes, it basically wil help in loading database into my dictionary from the DATABASE_URL environment variable.

Comment (vaidik) [inline requirements.txt:2]: Do you even know what this is used for?

Comment (swaraj7) [inline requirements.txt:2]: not totally sure, it helps in case of databases, related to environment variables that will be used by heroku .

**Pull request merged**